src/functions/api.ts

```diff
@@ -1,6 +1,10 @@
 import request from "graphql-request";
 import { CREATE_FEED_ITEM, UPDATE_FEED_ITEMS } from "../graphql/queries.js";
 
+const headers = {
+  authorization: "i-am-a-solution-function",
+};
+
```

Comment:
@paul-shannon-thrive-health — Heads up on another thing you'll have to update as part of your work. (Here and in `solution-template-monorepo`)